app/models/registrant.rb

```diff
@@ -1,7 +1,7 @@
 require 'roo'
 class Registrant < ApplicationRecord
   has_many :registrant_workshops
-  has_many :workshops, through: :registrant_workshops
+  belongs_to :workshops
```

Comment:
I think you'll need to look up where registrants are created and be sure they are properly assigned a workshop. Right now a RegistrantWorkshop is created, we'll have to get rid of that.

Reply:
Just fixed this error, but it's still not letting me add any registrants.  Let me know if I missed something (or if it's just my computer)

Reply:
@jhiatt do you get any errors? Pry open the session right before line 11 of the registrants_controller. Play around in there and see what's going on.